ci/test_wheel.sh

```diff
@@ -11,8 +11,6 @@ PYTHON_WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="cucim_${RAPIDS_PY_CUDA_SUFFIX}" rapids
 # echo to expand wildcard before adding `[extra]` requires for pip
 rapids-pip-retry install "$(echo ${PYTHON_WHEELHOUSE}/cucim*.whl)[test]"
 
-CUDA_MAJOR_VERSION=${RAPIDS_CUDA_VERSION:0:2}
-
```

Comment:
This unrelated-looking changed fixes `pre-commit` checks, which reported this:

```text
In ci/test_wheel.sh line 14:
CUDA_MAJOR_VERSION=${RAPIDS_CUDA_VERSION:0:2}
^----------------^ SC2034 (warning): CUDA_MAJOR_VERSION appears unused. Verify use (or export if used externally).
```

That variable was left behind by the changes in #903, and CI there reported the same error ([build link](https://github.com/rapidsai/cucim/actions/runs/16124571929/job/45498333365#step:7:194)), but that PR was admin-merged with that accidentally unfixed.

Reply:
Thanks James! 🙏

Ran into the same thing when doing more CUDA 11 removals in PR: https://github.com/rapidsai/cucim/pull/905

Given this is a common issue, have split that out in PR: https://github.com/rapidsai/cucim/pull/907

That will help us track the change (in case we learn there is some other script consuming this variable)

Reply:
Ok, sure. I didn't really think it was worth an entire separate CI run... the other change in this PR is so small and non-controversial, I'd expected it to get approved and merged quickly.